src/main/java/org/sonar/plugins/groovy/codenarc/CodeNarcProfileExporter.java

```diff
@@ -74,6 +76,17 @@ private void appendRule(ActiveRule activeRule) throws IOException {
       writer.append(AUTO_CLOSING_TAG);
     } else {
       writer.append("\">\n");
+      for (ActiveRuleParam activeRuleParam : activeRule.getActiveRuleParams()) {
+        String value = activeRuleParam.getValue();
+        String defaultValue = activeRuleParam.getRuleParam().getDefaultValue();
+        if (StringUtils.isNotBlank(value) && !value.equals(defaultValue)) {
+          writer.append("<property name=\"")
+            .append(activeRuleParam.getKey())
+            .append("\" value=\"")
+            .append(value)
```

Comment:
The value should be XML-escaped. For example, ampersand characters in the regex will lead to a corrupt CodeNarc profile.